gensite.js

```diff
@@ -85,3 +85,6 @@ sh.cp('-f', join(demoPath, 'bundle.js'), resolve(__dirname, 'dist', 'standard-de
 
 // copy favicons to dist
 sh.cp('-R', 'favicons', resolve(__dirname, 'dist'))
+
```

Comment:
Are these comments intended?

Reply:
Shoot. The first comment is intended, the second I forgot to uncomment after I checked backwards compatibility. Updating now.